manager/allocator/network/port/port.go

```diff
@@ -0,0 +1,470 @@
+package port
+
+import (
+	"fmt"
+
+	"github.com/docker/swarmkit/api"
+	"github.com/docker/swarmkit/manager/allocator/network/errors"
+)
+
+const (
+	// DynamicPortStart is the start of the dynamic port range from which node
+	// ports will be allocated when the user did not specify a port.
+	DynamicPortStart uint32 = 30000
+
+	// DynamicPortEnd is the end of the dynamic port range from which node
+	// ports will be allocated when the user did not specify a port.
+	DynamicPortEnd uint32 = 32767
```

Comment:
it is configurable, just at compile time instead of run time.

Reply:
No. I just thought a customer might hit this limit and it would be nice if we could tell them to change a setting rather than waiting for a new engine. But maybe they never get near to this limit anyway.